erpnext_documentation/www/docs/user/manual/en/manufacturing/bill-of-materials.md

```diff
@@ -106,7 +106,25 @@ The BOM can be linked to a [Project](/docs/user/manual/en/projects) to track pro
 
 The BOM can also be shown in the [Website](/docs/user/manual/en/website) for Open-source hardware products. Open-source hardware is similar to open-source where the product specifications are listed publicly.
 
-### 3.4 After Submitting
+### 3.4 BOM Template
+
+<img class="screenshot" alt="BOM Template" src="{{docs_base_url}}/assets/img/manufacturing/bom-template.png">
```

Comment:
screenshot width borders are too cropped, also show the top part. Be consistent with other screenshots.